addons/payment_paypal/tests/test_paypal.py

```diff
@@ -47,18 +53,23 @@ def _get_expected_values(self):
 
     def test_redirect_form_values(self):
         tx = self._create_transaction(flow='redirect')
-        with mute_logger('odoo.addons.payment.models.payment_transaction'):
-            processing_values = tx._get_processing_values()
+        with patch(
+            "odoo.addons.payment.utils.generate_access_token",
+            new=self._generate_test_access_token,
+        ):
+            with mute_logger('odoo.addons.payment.models.payment_transaction'):
+                processing_values = tx._get_processing_values()
 
         form_info = self._extract_values_from_html_form(processing_values['redirect_form_html'])
         self.assertEqual(
             form_info['action'],
             'https://www.sandbox.paypal.com/cgi-bin/webscr')
 
         expected_values = self._get_expected_values()
+        self.maxDiff = 1000
```

Comment:
Why is this required now?

Reply:
If I recall corectly it looks like something I forgot to remove.